docs/developers-landing.md

```diff
@@ -22,4 +13,13 @@ If you want to read top-viewed docs for developers, check out the following conc
 - [Pulsar APIs](pulsar-api-overview.md)
     - [Pulsar admin APIs](admin-api-overview.md)
     - [Pulsar REST APIs](reference-rest-api-overview.md)
-- [Pulsar contribution guide](../../contribute)
\ No newline at end of file
+- [Pulsar contribution guide](../../contribute)
+
+## Advanced topics
+
+If you want to learn how to develop Pulsar related components, check out the following guides.
+
+- [Develop simulation tools](develop-tools.md)

```

Comment:
To more accurately represent what is discussed in this doc, should this be titeld, "Create a test environment"?